unfoldingWord/translationCore
# recovered progress and checkmarks for menu on both tN & tW

### This pull request addresses:

- This PR makes sure that if data was generated for comments, selections, verseEdits and/ or bookmarks/reminders then the groupdata used for the menu to reflect that progress is matched with that data.

- Recover data persistence for menu glyph icons #58 https://github.com/translationCoreApps/translationNotes_Check_plugin/pull/58


### How to test this pull request:

- **you must pull feature/mc-1333 from tN**

- leave a comment then in the group data file in the filesystem change it from true to false. then reload the project.

- On tN add comments, selections, everseEdits and/or bookmarks/reminders and the menu should reflect those changes.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1387)
